arangopipe/makefile

```diff
@@ -0,0 +1,7 @@
+SRC = arangopipe/*.py
+python_arangopipe:$(SRC)
+	python3 setup.py sdist bdist_wheel
+clean:
+	rm -rf arangopipe.egg-info build dist
+	
+
```

Comment:
Unnecessary blank file.

Reply:
Not sure I follow. Which file is an uncessary blank file - the makefile? This builds the arangopipe python package.